cmd/util/suit.go

```diff
@@ -25,7 +25,8 @@ type Suit struct {
 	core.ClientCreator
 	// nemesis, seperated by comma.
 	Nemesises string
-
+	// perform service quality checking
+	WithProf   bool
```

Comment:
Is `WithProf` a common config field for all the cases?

Reply:
I add a field to control the schedule mode as `control.Config.Mode`, maybe we can use it later by add a `ModeWithProf` value.

Reply:
I wish it could be a common config field for all cases.

> I add a field to control the schedule mode as `control.Config.Mode`, maybe we can use it later by add a `ModeWithProf` value.

Good idea.